cfme/tests/pod/test_appliance_crud.py

```diff
@@ -13,29 +19,158 @@
 ]
 
 
+ansible_config = """
+# Create an OSEv3 group that contains the masters, nodes, and etcd groups
+[OSEv3:children]
+masters
+nodes
+etcd
+#
+# # Set variables common for all OSEv3 hosts
+[OSEv3:vars]
+# SSH user, this user should allow ssh based auth without requiring a password
+ansible_ssh_user=root
+
+# If ansible_ssh_user is not root, ansible_become must be set to true
+ansible_become=true
+openshift_deployment_type=openshift-enterprise
+
+# uncomment the following to enable htpasswd authentication; defaults to
+#DenyAllPasswordIdentityProvider
+openshift_master_identity_providers=[{{'name': 'htpasswd_auth', 'login': 'true', 'challenge': 'true', 'kind': 'HTPasswdPasswordIdentityProvider', 'filename': '/etc/origin/master/htpasswd'}}]
```

Comment:
I don't see you actually setting the login/password using htpasswd.

Reply:
it is already set. I just re-use openshift inventory config for cfme deployment. 
cfme inventory file also requires some data for correct deployment.